src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -2397,7 +2397,11 @@ namespace detail
              * on via JSON though.
              * Default is "1".
              */
-            m_IO.SetParameter( "StatsLevel", "0" );
+             m_IO.SetParameter( "StatsLevel", "0" );
+             auto StatLevel = auxiliary::getEnvNum( "OPENPMD_ADIOS2_STAT", 0 );
+
+             if (StatLevel > 0)
+                 m_IO.SetParameter( "StatsLevel", "1" );
```

Comment:
Please update the in-source comment a few lines above accordingly :)

For the code, I would suggest this logic change:
```suggestion
             auto stats_level = auxiliary::getEnvNum( "OPENPMD_ADIOS2_STATSLEVEL", 0 );
             m_IO.SetParameter( "StatsLevel", std::to_string( stats_level ) );
```
That way we can transparently pass through other levels in case they get added - the validation of numbers can be done by ADIOS2.

I would also rename `OPENPMD_ADIOS2_STAT` to `OPENPMD_ADIOS2_STATSLEVEL` to mirror the option